library/Icinga/Crypt/AesCrypt.php

```diff
@@ -276,4 +330,26 @@ private function encryptCBC($data)
 
         return base64_encode($hmac . $encrypt);
     }
+
+    /**
+     * whether the given method requires a tag
+     *
+     * @param string $method
+     *
+     * @return bool True if required false otherwise
+     */
+    public function isRequiredTag($method)
```

Comment:
I'd rather call this `isAuthenticatedEncryptionRequired()` (`isAESupported` would then be `isAuthenticatedEncryptionSupported()`). Also why is this requiring the method with a parameter? I can't see why it should not just use `getMethod()` internally.